diff --git a/node_modules/@opensumi/ide-electron-basic/lib/browser/index.js b/node_modules/@opensumi/ide-electron-basic/lib/browser/index.js
index e5d8d58..5490699 100644
--- a/node_modules/@opensumi/ide-electron-basic/lib/browser/index.js
+++ b/node_modules/@opensumi/ide-electron-basic/lib/browser/index.js
@@ -53,22 +53,22 @@ const nativeRoles = [
     },
     {
         name: 'copy',
-        key: 'ctrlcmd+c',
+        key: 'ctrlcmd+shift+c',
         when: '!editorFocus',
     },
     {
         name: 'paste',
-        key: 'ctrlcmd+v',
+        key: 'ctrlcmd+shift+v',
         when: '!editorFocus',
     },
     {
         name: 'selectAll',
-        key: 'ctrlcmd+a',
+        key: 'ctrlcmd+shift+a',
         when: '!editorFocus',
     },
     {
         name: 'cut',
-        key: 'ctrlcmd+x',
+        key: 'ctrlcmd+shift+x',
         when: '!editorFocus',
     },
     {
